includes/data-stores/abstract-wc-order-data-store-cpt.php

```diff
@@ -69,7 +69,7 @@ public function create( &$order ) {
 					'post_type'     => $order->get_type( 'edit' ),
 					'post_status'   => 'wc-' . ( $order->get_status( 'edit' ) ? $order->get_status( 'edit' ) : apply_filters( 'woocommerce_default_order_status', 'pending' ) ),
 					'ping_status'   => 'closed',
-					'post_author'   => 1,
+					'post_author'   => is_callable( array( $order, 'get_customer_id' ) ) ? $order->get_customer_id() : 1,
```

Comment:
The default "1" corresponds to admin user. Should we try to force 0 for guest orders?

Reply:
@mikejolley after this change, `1` will be used only for refund orders which it seems to me that is fine since refunds don't have a `_customer_user` post meta. `0` will be used for regular orders placed by guest users. Am I missing something?

Reply:
I think we need to keep it consistent throughout order types and always set it to the user who owns the order/refund? I worry that using 1 could mean someone using this field would get extra rules when getting admin (user 1) orders.